pkg/aws_config_client/completer.go

```diff
@@ -227,16 +236,47 @@ func (c *completer) writeAWSProfiles(out *ini.File, region string, profiles []*A
 		out.DeleteSection(profileSection)
 		section, err := out.NewSection(profileSection)
 		if err != nil {
-			return errors.Wrapf(err, "Unable to create %s section in AWS Config", profileSection)
+			return nil, errors.Wrapf(err, "Unable to create %s section in AWS Config", profileSection)
 		}
 		section.Key(AWSConfigSectionOutput).SetValue("json")
 		section.Key(AWSConfigSectionCredentialProcess).SetValue(credsProcessValue)
 		section.Key(AWSConfigSectionRegion).SetValue(region)
 	}
-	return nil
+	return out, nil
 }
 
-func (c *completer) Complete(out *ini.File) error {
+func (c *completer) mergeConfigs(newAWSProfiles *ini.File, base *ini.File) (*ini.File, error) {
+
+	// Ask user to confirm that this is the AWS config they want
+	newAWSProfiles.WriteTo(os.Stdout)
+	cnt, err := c.prompt.Confirm("Does this config file look right?", true)
+	if !cnt {
+		logrus.Info("Discarding changes")
+		return ini.Empty(), nil
+	}
+
+	baseBytes := bytes.NewBuffer(nil)
+	newAWSProfileBytes := bytes.NewBuffer(nil)
+	writeCode, err := newAWSProfiles.WriteTo(newAWSProfileBytes)
+	if err != nil {
+		logrus.Errorf("Discarding changes. Unable to write AWS Profiles: %v, writecode: %d", err, writeCode)
+		return ini.Empty(), err
+	}
+	writeCode, err = base.WriteTo(baseBytes)
+	if err != nil {
+		logrus.Errorf("Discarding changes. Unable to incorporate original AWS config file: %v, writecode: %d", err, writeCode)
```

Comment:
feel like instead of logrus error here, we should` errors.Wrapf(err, ""....)` this will eventually make its way to a log statement.